utils/bashrc.d/14-kube-ps1.bashrc

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
-source ${HOME}/.bashrc.d/05-kube-ps1.sh
-
-export PS1="[\W {\[\033[1;32m\]${OCMC_OCM_URL}\[\033[0m\]} \$(kube_ps1)]\$ "
+export PS1="[\W {\[\033[1;32m\]\$(ocm config get url)\[\033[0m\]} \$(kube_ps1)]\$ "
```

Comment:
Ensures the env is properly displayed each time the terminal line updates, so it's always accurate and doesn't require re-sourcing .bashrc

Reply:
Can we show short env in PS1`[{prod}]` ? my only concern is that it will show lengthy env like `[{https://api.openshift.com}]`

Reply:
We could always overwrite individually in the personalizations mount - IIRC those are automatically sourced at the end of the source change so you could do something like:

```
ocm_short_url() {
  env_url=$(ocm config get url)
  if grep staging <<< $env_url; then
    echo "staging"
    return
  elif grep integration <<< $env_url; then
    echo "integration"
    return
  fi
  echo "production"
 }
 
 export PS1="[\W {\[\033[1;32m\]\$(ocm_short_urll)\[\033[0m\]} \$(kube_ps1)]\$"
 ```
 
 Written entirely from memory but the idea is that would allow you to customize your own PS1 to show whatever you wanted it to.

Reply:
Can we get the short url from ocm somehow so we don't need ocm-container specific code to do it?
Or push our changes upstream?

Reply:
FWIW, this had to be moved back to ${OCM_URL}, as the `ocm config get url` no longer necessarily gets the correct URL, as we're overriding the ocm config URL with the env var.

This will still be a long-ish URL though, but also no different from what is currently displayed in OCM Container, I believe.

I'm happy for any other commits to make this cleaner in the future though.